pkg/ingester/client/compat.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+const (
+	offset64 = 14695981039346656037
```

Comment:
nit: Even it's forked, do you think it's worth adding some comment on what `offset64` and `prime64` actually are?

Reply:
Sure! I pushed a commit that adds a comment documenting them, WDYT?